### PR TITLE
EX04 is done

### PR DESCRIPTION
**Pull request by eithangav**
my solution for EX04 (please ignore the fact that the branch's name is EX03 :) )

### CODE REVIEW

**Comment by eithangav** [comment]
my code files:
server.js
data.json

File: EX04/server.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+let app = express();
+let port = 8080;
+
+//parsing
+app.use(bodyParser.json());
+
+//request: show all current tasks
+app.get('/tasks', (req, res) => {
+	//read the json
+	let data = fs.readFileSync('data.json', 'utf8');
```
Comment (talboren) [inline EX04/server.js:14]: Read the file on server start instead on every request

File: EX04/server.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	let strData = JSON.parse(data);
```
Comment (talboren) [inline EX04/server.js:15]: variable name is misleading

**Review by talboren — APPROVED** [review]
Good overall job, add node_modules/* to .gitignore file

**Pull request merged**